how-to/what-is-vm-cache-and-how-do-I-use-it.md

```diff
@@ -0,0 +1,31 @@
+# What Is VMCache and How To Enable It
+
+### What is VMCache
+VMCache is a new function that creates VMs as caches before using it.
+It helps speed up new container creation.
+The function consists of a server and some clients communicating
+through Unix socket.  The protocol is GRPC in protocols/cache/cache.proto.
+The VMCache server will create some VMs and cache them by factory cache.
+It will convert the VM to GRPC format and transport it when gets
+requestion from clients.
+Factory grpccache is the VMCache client.  It will request GRPC format
+VM and convert it back to a VM.  If VMCache function is enabled,
+kata-runtime will request VM from factory grpccache when it creates
+a new sandbox.
+
+### How to enable VMCache
+VMCache can be enabled by changing your Kata Containers config file (`/usr/share/defaults/kata-containers/configuration.toml`,
+overridden by `/etc/kata-containers/configuration.toml` if provided) such that:
+* `vm_cache_number` the number of caches of VMCache:
+    * unspecified or == 0   --> VMCache is disabled
+    * `> 0`                   --> will be set to the specified number
+*  `vm_cache_endpoint` specifies the address of the Unix socket.
+
+Then you can create a VM templating for later usage by calling
+```
+$ sudo kata-runtime factory init
+```
+and purge it by ctrl-c it.
+
+Currently, VM cache still cannot work with VM templating and vsock.
```

Comment:
I'd make this into a new section. Something like:

> # Limitations
>
> Restrictions on the use of VM cache:
>
> - Cannot work with VM templating.
> - Cannot work with vsock.
> - Only supports the qemu hypervisor.

Reply:
> I'd make this into a new section. Something like:
> 
> > # Limitations
> > Restrictions on the use of VM cache:
> > 
> > * Cannot work with VM templating.
> > * Cannot work with vsock.
> > * Only supports the qemu hypervisor.

Added.